vignettes/transition_disruption_metric.Rmd

```diff
@@ -0,0 +1,552 @@
+            1, 2020L,               "S(t)",    "solid",
+            2, 2025L,               "S(t)",    "solid",
+            3, 2030L,               "S(t)",    "solid",
+            4, 2035L,               "S(t)",    "solid",
+            5, 2040L,               "S(t)",    "solid",
+            6, 2045L,               "S(t)",    "solid",
+            7, 2050L,               "S(t)",    "solid",
+          0.5, 2020L,               "P(t)",    "solid",
+            2, 2025L,               "P(t)",    "solid",
+            2, 2025L, "Required build-out",   "dashed",
+            3, 2030L, "Required build-out",   "dashed"
+  )
+
+data %>%
+  ggplot(aes(x = year, y = production, group = type)) +
+  geom_line(aes(color = type, linetype = line_type)) +
+  scale_color_manual(values = c("#9999CC", "#0B7A75", "#CC6666")) +
+  guides(linetype = "none") +
+  scale_linetype_manual(values = c("dashed", "solid")) + 
+  guides(fill = "none") +
+  geom_point(aes(shape = type)) +
+  scale_shape_manual(values = c("triangle", "triangle", "circle")) +
+  labs(title = "Production by Year", x = "Year", y = "Production") + 
+  theme(
+    axis.text.y = element_blank()
+  ) + 
+  scale_x_continuous(
+    breaks = c(2020, 2025, 2030, 2035, 2040, 2045, 2050),
+    labels = c("t", "t+t1", "t+t2", "", "", "", "")
+    )
+```
+
+- Case 3: $0 < \frac{m_P}{m_S} < 1$: The portfolio is producing more than the scenario at $t+t_1$, but still less than is required at $t+t_2$. The slope of required build-out will be positive, but less than the slope of the scenario. 
+
+```{r, echo = FALSE}
+
+data <- tibble::tribble(
+  ~production, ~year,                ~type, ~line_type,
+            1, 2020L,               "S(t)",    "solid",
+            2, 2025L,               "S(t)",    "solid",
+            3, 2030L,               "S(t)",    "solid",
+            4, 2035L,               "S(t)",    "solid",
+            5, 2040L,               "S(t)",    "solid",
+            6, 2045L,               "S(t)",    "solid",
+            7, 2050L,               "S(t)",    "solid",
+          0.5, 2020L,               "P(t)",    "solid",
+            2.5, 2025L,               "P(t)",    "solid",
+            2.5, 2025L, "Required build-out",   "dashed",
+            3, 2030L, "Required build-out",   "dashed"
+  )
+
+data %>%
+  ggplot(aes(x = year, y = production, group = type)) +
+  geom_line(aes(color = type, linetype = line_type)) +
+  scale_color_manual(values = c("#9999CC", "#0B7A75", "#CC6666")) +
+  guides(linetype = "none") +
+  scale_linetype_manual(values = c("dashed", "solid")) + 
+  guides(fill = "none") +
+  geom_point(aes(shape = type)) +
+  scale_shape_manual(values = c("triangle", "triangle", "circle")) +
+  labs(title = "Production by Year", x = "Year", y = "Production") + 
+  theme(
+    axis.text.y = element_blank()
+  ) + 
+  scale_x_continuous(
+    breaks = c(2020, 2025, 2030, 2035, 2040, 2045, 2050),
+    labels = c("t", "t+t1", "t+t2", "", "", "", "")
+    )
```

Comment:
Run addin "Style active file" (pop up the commands pallete with Shift + Ctrl/Cmd + P).